Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -24,6 +25,8 @@ fileprivate let placesFetchMonitorIdentifier = "PlaceFetchRadiusMonitor"
 
 class PlaceCarouselViewController: UIViewController {
 
+    lazy var placesWithEvents = Set<Place>()
```

Comment:
leftovers from things that I tried to fix the non-interaction of details cards on display. Will remove